secrethub/provider_test.go

```diff
@@ -16,6 +16,8 @@ const (
 	envNamespace         = "SECRETHUB_TF_ACC_NAMESPACE"
 	envRepo              = "SECRETHUB_TF_ACC_REPOSITORY"
 	envSecondAccountName = "SECRETHUB_TF_ACC_SECOND_ACCOUNT_NAME"
+	envAWSKMSKey         = "SECRETHUB_TF_ACC_AWS_KMS_KEY"
```

Comment:
We had the idea to instead generate those right? (if AWS doesn't price us for it every time)

Reply:
We would be charged for every key, which is why we’ve done it like this.

Reply:
Doesn't seem to be the case https://aws.amazon.com/kms/pricing/. Deleting = deleting, billing stops, unless you cancel the deletion.

Reply:
Billing seems to be per month. Doesn't that mean that you'd be charged for 1 month for every created key? That'd mean €2,- per test run.

Reply:
Discussed this in person and we've decided to use the KMS key managed outside this project.